src/webparts/sendToLaserficheLoginComponent/components/SendToLaserficheLoginComponent.tsx

```diff
@@ -661,29 +324,37 @@ export default class SendToLaserficheLoginComponent extends React.Component<
       };
 
       const entryCreateResult: CreateEntryResult =
-        await this.repoClient.entriesClient.importDocument(entryRequest);
+        await repoClient.entriesClient.importDocument(entryRequest);
       const Entryid14 = entryCreateResult.operations.entryCreate.entryId;
-      filelink = `https://app.${this.state.region}/laserfiche/DocView.aspx?db=${repoId}&docid=${Entryid14}`;
-
-      document.getElementById('it').innerHTML = 'Document uploaded';
-      document.getElementById('imgid').style.display = 'none';
-      document.getElementById('divid').style.display = 'block';
-      document.getElementById('divid1').onclick = this.Dc;
-      document.getElementById('divid13').style.display = 'block';
-      document.getElementById('divid13').onclick = this.viewfile;
-      document.getElementById('divid14').onclick = this.Redirect;
-      window.localStorage.removeItem(TempStorageKeys.LContType);
+      const fileLink = getEntryWebAccessUrl(
+        Entryid14.toString(),
+        repoId,
+        webClientUrl,
+        false
+      );
+      const pageOrigin = spFileMetadata.pageOrigin;
+      const fileUrl = spFileMetadata.fileUrl;
+      const fileUrlWithoutDocName = fileUrl.slice(0, fileUrl.lastIndexOf('/'));
+      const path = pageOrigin + fileUrlWithoutDocName;
+
+      await dialog.close();
+      dialog.fileLink = fileLink;
+      dialog.pathBack = path;
+      dialog.isLoading = false;
+      dialog.metadataSaved = true;
+      dialog.show();
+      window.localStorage.removeItem('spdocdata');
     } catch (error) {
       window.alert(`Error uploding file: ${JSON.stringify(error)}`);
-      window.localStorage.removeItem(TempStorageKeys.LContType);
+      window.localStorage.removeItem('spdocdata');
```

Comment:
should we should an error to the user if the call failed for some reason?
how about receiving a 409?

Reply:
window.alert is notifying user, but this should be standardized